pkgs/by-name/re/rerun/package.nix

```diff
@@ -31,7 +32,17 @@
     "map_view"
   ],
 }:
+let
+  clang-unwrapped = llvmPackages.clang-unwrapped;
+  libclang = llvmPackages.libclang.lib;
```

Comment:
Why setting `libclang = libclang.lib` and but `clang-unwrapped = clang-unwrapped`? Maybe make it more consistent?